moabb/datasets/__init__.py

```diff
@@ -54,3 +54,24 @@
 # Call this last in order to make sure the dataset list contains all
 # the datasets imported in this file.
 _init_dataset_list()
```

Comment:
no problem on my side!

We just have to remove the test that ensures that the dataset codes are unique in the list.

Maybe we could also put the compound datasets in the same list instead of having a separate `compound_datasets_list`

Reply:
Nice! I will handle with compound datasets in another PR.